lib/ansible/modules/cloud/amazon/ec2_eip.py

```diff
@@ -360,6 +376,7 @@ def main():
         reuse_existing_ip_allowed=dict(required=False, type='bool',
                                        default=False),
         release_on_disassociation=dict(required=False, type='bool', default=False),
+        allow_reassociation=dict(required=False, type='bool', default=False),
```

Comment:
`required=False` is not required (I know other lines have it but they're wrong ;) )

Reply:
:D Ok, removed it.
(I suspected that `required` was not... required ; but assumed the convention was to be explicit about it)